### PR TITLE
Vesuvius -> Yellowstone

### PR DESCRIPTION
**Pull request by awisniew207**
- Chronicle Vesuvius -> Chronicle Yellowstone
- Renamed files which specified `vesuvius`
- Fixed broken links I found when renaming them to Yellowstone
- Double-checked all links I renamed for functionality 
- Changed chain information to match what we have [here](https://app.conduit.xyz/view-network/chronicle-yellowstone-testnet-9qgmzfcohk/overview) 

My questions:
`./migrations/migrating-to-yellowstone` did not exist, changed it to `./migrating-to-yellowstone`. Two instances of this, should we make a new folder for it? Current location is `docs/connecting-to-a-lit-network`.
`docs/connecting-to-a-lit-network/testnets.md` says 6.1.0 is the earliest version of `datil-dev`, with the migration to Yellowstone, what would we change it to?

### CODE REVIEW

**Comment by a1ttech** [comment]
> ./migrations/migrating-to-yellowstone did not exist, changed it to ./migrating-to-yellowstone. Two instances of this, should we make a new folder for it? Current location is docs/connecting-to-a-lit-network.

Right now we have this [page](https://developer.litprotocol.com/connecting-to-a-lit-network/migrating-to-vesuvius) in the docs which has the URL 'migrating-to-vesuvius' , we should probably update that right?

**Comment by a1ttech** [comment]
Or actually it looks like you already did that. I think you're asking where the link should link to? Because i think we should link to this page: https://developer.litprotocol.com/connecting-to-a-lit-network/migrating-to-vesuvius

**Comment by a1ttech** [comment]
I believe latest would be 6.2.2 (not yet published) but not 100% sure

**Comment by spacesailor24** [comment]
@awisniew207 I'm going to change base branch of this PR to `wyatt/datil-test` as they both should be merged together and `wyatt/datil-test` needs the `vesuvius -> yellowstone` changes as well

**Comment by spacesailor24** [comment]
> docs/connecting-to-a-lit-network/testnets.md says 6.1.0 is the earliest version of datil-dev, with the migration to Yellowstone, what would we change it to?

I think it needs to be the latest version of the SDK (the one that introduces yellowstone support)

File: src/components/AddRollupButtonYellowstone.jsx [hunk 1]
```diff
@@ -4,16 +4,16 @@ export default function AddRollupButton() {
   const addNetwork = () => {
     const params = [
       {
-        chainId: '0x907',
-        chainName: 'Chronicle Vesuvius - Lit Protocol Testnet',
+        chainId: '175188',
```
Comment (spacesailor24) [inline src/components/AddRollupButtonYellowstone.jsx:7]: Are we sure this can be an int? Did you testing adding it to MetaMask?

**Comment by spacesailor24** [comment]
Other than the merge conflicts I made, lgtm

File: src/components/AddRollupButtonYellowstone.jsx (hunk 1 above)
Comment (awisniew207) [reply src/components/AddRollupButtonYellowstone.jsx:7]: Yeah manually added it using an int and not hex, works 👍  tested with the faucet and got tokens to the chain

**Comment by awisniew207** [comment]
Datil -> datil-prod
Any 6.1.0/6.2.0 for minimum version updated to 6.2.2 to support Yellowstone
Removed all Vesuvius again
Checked the links again

File: docs/connecting-to-a-lit-network/migrating-to-datil.md [hunk 2]
```diff
@@ -5,38 +5,34 @@ import TabItem from '@theme/TabItem';
 
 Lit is launching three new networks, Datil-dev, Datil-test, and Datil, to improve performance and stability for production users compared to previous Lit networks like Cayenne, Manzano, and Habanero.
 
-The Datil and Datil-test networks use a new rollup blockchain called [Chronicle Yellowstone](./lit-blockchains/chronicle-yellowstone.md), and Datil-dev uses a new rollup blockchain called [Chronicle Vesuvius](./lit-blockchains/chronicle-vesuvius); Both new blockchains replace the [Chronicle](./lit-blockchains/chronicle) blockchain that powered Lit's earlier networks. You will learn how to migrate data between these chains below.
+The Datil-dev and Datil-test networks use a new rollup blockchain called [Chronicle Yellowstone](./lit-blockchains/chronicle-yellowstone.md). This new blockchains replaces the [Chronicle](./lit-blockchains/chronicle) blockchain that powered Lit's earlier networks. You will learn how to migrate data between these chains below.
```
Comment (spacesailor24) [inline docs/connecting-to-a-lit-network/migrating-to-datil.md:8]:
```suggestion
The Datil-dev and Datil-test networks use a new rollup blockchain called [Chronicle Yellowstone](./lit-blockchains/chronicle-yellowstone.md). These new blockchains replace the [Chronicle](./lit-blockchains/chronicle) blockchain that powered Lit's earlier networks. You will learn how to migrate data between these chains below.
```

File: docs/connecting-to-a-lit-network/migrating-to-datil.md [hunk 3]
```diff
@@ -61,11 +57,11 @@ For latest on which Datil networks are available to connect to, please refer to
 The only code changes required to make use of the new Datil networks are as follows:
 
 - Upgrade the Lit packages to the latest version that supports Datil
-  - `6.2.0` is the minimum version of the packages that supports `datil-test`
-  - `6.1.0` is the minimum version of the packages that supports `datil-dev`
+  - `6.2.2` is the minimum version of the packages that supports `datil-test`
+  - `6.2.2` is the minimum version of the packages that supports `datil-dev`
```
Comment (spacesailor24) [inline docs/connecting-to-a-lit-network/migrating-to-datil.md:61]:
```suggestion
  - `6.2.2` is the minimum version of the packages that support `datil-test` and `datil-dev`
```

**Review by spacesailor24 — APPROVED** [review]

**Pull request merged**